r/vignettes/data_types.Rmd

```diff
@@ -96,26 +122,38 @@ Note that the Arrow specification also permits half-precision (16 bit) floating
 
 ## Fixed point decimal types
 
-Arrow also contains `decimal()` data types, in which numeric values are specified in decimal format rather than binary. Decimals in Arrow come in two varieties, a 128 bit version and a 256 bit version, but in most cases users should be able to use the more general `decimal()` data type function rather than the specific `decimal128()` and `decimal256()` functions. 
+Arrow also contains `decimal()` data types, in which numeric values are specified in decimal format rather than binary.
+Decimals in Arrow come in two varieties, a 128 bit version and a 256 bit version, but in most cases users should be able 
+to use the more general `decimal()` data type function rather than the specific `decimal32()`, `decimal64()`, `decimal128()`,
```

Comment:
I split the text up in this vignette to make it easier to edit and track edits in future, but the only actual changes I made were to this line, where I added the new decimal types in.